frontend/src/utils/index.js

```diff
@@ -352,8 +352,8 @@ export function getTimeStringTo (time, toTime, withoutPrefix = false) {
   return moment(time).to(toTime, withoutPrefix)
 }
 
-export function isOwnSecret (infrastructureSecret) {
-  return get(infrastructureSecret, ['metadata', 'secretRef', 'namespace']) === get(infrastructureSecret, ['metadata', 'namespace'])
+export function isOwnSecret (secretBinding) {
```

Comment:
rename to `refersToOwnSecret` or `hasOwnSecret`?

Reply:
renamed to `hasOwnSecret`